plugins/base/src/main/resources/dokka/scripts/symbol-parameters-wrapper_deferred.js

```diff
@@ -0,0 +1,63 @@
+// helps with some corner cases where <wbr> starts working already,
+// but the signature is not yet long enough to be wrapped
+const leftPaddingPx = 60
+
+const wrapAllSymbolParameters = () => {
+    document.querySelectorAll("div.symbol").forEach(symbol => wrapSymbolParameters(symbol))
+}
+
+const wrapSymbolParameters = (symbol) => {
+    let symbolBlockWidth = symbol.clientWidth
+    let innerTextWidth = Array.from(symbol.children)
```

Comment:
Should it ignore annotations? 
![image](https://user-images.githubusercontent.com/9636937/189687333-740c3d0a-9c96-4ff7-a47e-01eb78631ca3.png)

Reply:
Yeah, good point! it should probably ignore all blocks

also on your screnshot it looks like there's 8 spaces instead of 4 :thinking:

Reply:
Sorry, it happened due to my debugger.